ymer/src/main/java/com/avanza/ymer/PersistedInstanceIdCalculationServiceMBean.java

```diff
@@ -21,4 +21,11 @@
 
 	void calculatePersistedInstanceId(String collectionName);
 
+	/**
+	 * Returns number of partitions that all collections enabled for persisted instance id are currently
+	 * ready for.
+	 * When only one collection is used, this is the same as {@link PersistedInstanceIdStatisticsMBean#getReadyForNumberOfPartitions()}.
+	 */
+	int[] getAllCollectionsReadyForNumberOfPartitions();
```

Comment:
Maybe it's subjective, but the name here is hard to parse for me. Would it be confusing if this was also called `getReadyForNumberOfPartitions`, or something like "get number of partitions that the data is prepared for"?

Reply:
I agree, I changed this name to `getNumberOfPartitionsThatDataIsPreparedFor` and for the collections it's called `getNumberOfPartitionsThatCollectionIsPreparedFor`.